addons/website/static/src/scss/website.scss

```diff
@@ -1951,6 +1951,21 @@ $ribbon-padding: 100px;
     z-index: $zindex-modal + 1; // Over the modal backdrop.
 }
 
+// Search
+.o_searchbar_form {
+    input::-webkit-search-cancel-button  {
+        -webkit-appearance: none;
+        content: escape-svg($btn-close-bg);
+        height: $input-font-size;
+        cursor: pointer;
+    }
```

Comment:
Is that something you want specificaly for `o_searchbar_form` element only?

Reply:
I initially applied it on the `o_searchbar_form` because I wrongly assumed the input type search to always come with the searchbar template, but indeed it's harmless to apply it on a more generic selector as it's just about changing the default browser `X` clear input design.

I'm going to apply it to `input[type="search"]` since -webkit-search-cancel-button is relevant on input type=search.